applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -18,24 +17,51 @@
 
 #include "macros_common.h"
 #include "le_audio.h"
-#include "nrf5340_audio_common.h"
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(broadcast_source, CONFIG_BLE_LOG_LEVEL);
 
 BUILD_ASSERT(CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT <= 2,
 	     "A maximum of two audio streams are currently supported");
 
-ZBUS_CHAN_DEFINE(le_audio_chan, struct le_audio_msg, NULL, NULL, ZBUS_OBSERVERS_EMPTY,
-		 ZBUS_MSG_INIT(0));
-
-ZBUS_CHAN_DEFINE(sdu_ref_chan, struct sdu_ref_msg, NULL, NULL, ZBUS_OBSERVERS_EMPTY,
-		 ZBUS_MSG_INIT(0));
-
 #define STANDARD_QUALITY_16KHZ 16000
 #define STANDARD_QUALITY_24KHZ 24000
 #define HIGH_QUALITY_48KHZ     48000
 
+#if CONFIG_BT_AUDIO_BROADCAST_CONFIGURABLE
+#define BT_BAP_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                  \
```

Comment:
Maybe we don't need  to use #define at all.
We could potentially do "static struct bt_bap_lc3_preset lc3_preset = (...)" directly